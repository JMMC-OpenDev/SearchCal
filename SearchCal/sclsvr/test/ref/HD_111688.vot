version https://git-lfs.github.com/spec/v1
oid sha256:ddfdedd4abe0679de09d99d46a953d02c1db8463384e9841c86929367a83d24a
size 278359
