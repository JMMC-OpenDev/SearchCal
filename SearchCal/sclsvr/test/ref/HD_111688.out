version https://git-lfs.github.com/spec/v1
oid sha256:fc3e7ba34c6a8dfde23c620bde1fb61a04107b285c0e5d4bf42c355f49287031
size 283998
